src/utils/helpers.ts

```diff
@@ -134,9 +134,35 @@ export function isChecked(value: any) {
 }
 
 export const isMobile = () => {
-  return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
-    navigator.userAgent,
-  );
+  let check = false;
```

Comment:
The PR itself looks good but have you considered using some library to detect mobile devices, e.g. https://www.npmjs.com/package/react-device-detect ? Or where is this regex coming from?